applets/005-netperf/applet.py

```diff
@@ -30,20 +30,20 @@ def netserver_start(x, host_name, host_ip, host_user, netserver_port):
 def main(x, conf, args):
 
     if not len(args) == 7:
-        x.p.msg("wrong usage. use: source dest id:[id] sport:[port] "
-                "dport:[port] length:[bytes|seconds] netserver:[port]\n")
+        x.p.msg("wrong usage. use: source:[name] dest:[name] id:[id] sport:["
```

Comment:
argl: I suggest the change solely for dest - not source! Why? Because a source must always provided and is aligned with *all* other applets. Sorry that I do not express myself clear enough, sorry.

I think we **should** rename the word source into something else. It is always the host where the applet is executed ... Mhhh, do you have any ideas how to name this?